comps/3rd_parties/tgi/deployment/kubernetes/gaudi-values.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for tgi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+accelDevice: "gaudi"
+
+image:
+  repository: ghcr.io/huggingface/tgi-gaudi
+  tag: "2.0.6"
+
+MAX_INPUT_LENGTH: "1024"
+MAX_TOTAL_TOKENS: "2048"
+CUDA_GRAPHS: ""
+OMPI_MCA_btl_vader_single_copy_mechanism: "none"
+ENABLE_HPU_GRAPH: "true"
+LIMIT_HPU_GRAPH: "true"
+USE_FLASH_ATTENTION: "true"
+FLASH_ATTENTION_RECOMPUTE: "true"
+
+resources:
+  limits:
+    habana.ai/gaudi: 1
```

Comment:
This could also mention model, and corresponding requests for CPU & memory, to get better priority for the pod, and to avoid too much of CPU pods being put on Gaudi node (as taints are not used to keep non-device workloads out of it).